Eatery Blue/UI/EateryFilter/EateryFilterViewController.swift

```diff
@@ -16,12 +16,12 @@ protocol EateryFilterViewControllerDelegate: AnyObject {
 
 class EateryFilterViewController: UIViewController {
 
-    let under10Minutes = PillFilterButtonView()
-    let paymentMethods = PillFilterButtonView()
-    let favorites = PillFilterButtonView()
     let north = PillFilterButtonView()
     let west = PillFilterButtonView()
     let central = PillFilterButtonView()
+    let under10Minutes = PillFilterButtonView()
+    let paymentMethods = PillFilterButtonView()
+    let favorites = PillFilterButtonView()
```

Comment:
Private? Yea this Eatery codebase does not utilize encapsulation well :(

Ideally, we should sort these alphabetically but not a big deal

Reply:
i was thinking order they appear